docs/src/main/resources/timestamps.html

```diff
@@ -1,160 +0,0 @@
-<!--
-  Licensed to the Apache Software Foundation (ASF) under one or more
-  contributor license agreements.  See the NOTICE file distributed with
-  this work for additional information regarding copyright ownership.
-  The ASF licenses this file to You under the Apache License, Version 2.0
-  (the "License"); you may not use this file except in compliance with
-  the License.  You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
--->
-<html>
```

Comment:
For every HTML file, similar content existed in user manual.  However, there were a few cases where not all text HTML doc was not moved to user manual.  I tried to add this text.  However, I did not add old JMX metrics docs in metrics.html as it looks like hadoop metrics is preferred.  Also, administration.html could be checked over too.  While it looks like all docs were added to manual, there was a lot of documentation so it was not easy to confirm.